charts/kube-prometheus-stack/templates/prometheus/ingress.yaml

```diff
@@ -18,7 +18,7 @@ metadata:
   namespace: {{ template "kube-prometheus-stack.namespace" . }}
   labels:
     app: {{ template "kube-prometheus-stack.name" . }}-prometheus
-{{ include "kube-prometheus-stack.labels" . | indent 4 }}
+{{- include "kube-prometheus-stack.labels" . | indent 4 }}
```

Comment:
Why here and not elsewhere? This exists in 100+ places in this chart

Reply:
Ah yeah, I just realized after making this PR. I can go ahead and modify everything else if you'd like.

Reply:
Sure, might as well clean this up a little. Would want to make sure we don't accidentally remove the line break as well.